docs/table/tips_and_tricks.rst

```diff
@@ -0,0 +1,42 @@
+Tips and Tricks
+***************
+
+Using QTable to store user-defined units in a FITS file
+=======================================================
+
+A `~astropy.table.QTable` can be used to store data as :ref:`user-defined
```

Comment:
```suggestion
A |QTable| can be used to store data as :ref:`user-defined
```
Not all of the existing documentation has been updated to take advantage of the [available ReST substitutions](https://github.com/astropy/astropy/blob/main/docs/common_links.txt), but new documentation should use them.